### ISSUE TITLE
Accessibility: buttons are not focusable with Tab key

### ISSUE THREAD

**Issue by lukaw3d** [oasisprotocol/oasis-wallet-ext#74]
In this UI only "Send" and "Receive" are focusable with Tab key:
![btns](https://user-images.githubusercontent.com/3758846/130175828-f206466b-b8fd-4ea9-913e-5c953dfdd4ab.png)
